app/api/views/users.py

```diff
@@ -173,6 +173,9 @@ def signup():
     user = User.query.filter(
         User.email_address == email_address.lower()).first()
 
+    if '<' in email_address or '>' in email_address or email_address.count('@') > 1:
```

Comment:
does this work?
```suggestion
    if ['<', '>'] in email_address or email_address.count('@') > 1:
```

Reply:
Doesn't look like it - `TypeError: 'in <string>' requires string as left operand, not list`